<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReduceSubProductDialog</class>
 <widget class="QDialog" name="ReduceSubProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reduce amount of subproducts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="reduceL">
     <property name="text">
      <string>Amount to reduce</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="reduceSpinBox"/>
   </item>
   <item>
    <widget class="QLabel" name="reasonL">
     <property name="text">
      <string>Reason</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="reasonComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="subProdLeft">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReduceSubProductDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReduceSubProductDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
